test/api/v3/integration/groups/POST-groups_groupId_join.js

```diff
@@ -0,0 +1,63 @@
+import {
+  generateUser,
+} from '../../../../helpers/api-integration.helper';
+
+describe('POST /group/:groupId/join', () => {
+  let user;
+
+  beforeEach(() => {
+    return generateUser().then((generatedUser) => {
+      user = generatedUser;
+    });
+  });
+
+  context('Guilds', () => {
+    it('inviting to guild doesn\'t give basilist to inviter', () => {
+      let groupName = 'Test Guild';
+      let groupType = 'guild';
+
+      return generateUser({balance: 1, 'items.quests.basilist': 0 }).then((generatedUser) => {
+        user = generatedUser;
+        return user.post('/groups', {
+          name: groupName,
+          type: groupType
+        });
+      })
+      .then((guild) => {
+        return generateUser({ 'invitations.guilds': [{ id: guild._id}]}).then((generatedUser) => {
+          return generatedUser.post('/groups/' + guild._id + '/join');

```

Comment:
in this case it was done because we don't have the invite route ready yet, in general I don't know. To me it doesn't look like calling real routes makes things more difficult, rather it should help with routes that have a lot of side effects that are more difficult to replicate. But I'll leave the decision to @crookedneighbor